apps/cyberstorm-remix/app/root.tsx

```diff
@@ -51,10 +28,12 @@ import {
 } from "cyberstorm/security/publicEnvVariables";
 import { StorageManager } from "@thunderstore/ts-api-react/src/storage";
 import { Route } from "./+types/root";
+import { RootLayout } from "./rootRoute/rootLayout";
```

Comment:
Nitpick: If we want to keep the RootLayout in a separate file, this could be just `export { RootLayout as Layout } from "./rootRoute/rootLayout";`